consumer/src/test/kotlin/integrations/kafka/KafkaEventSinkSimple.kt

```diff
@@ -0,0 +1,93 @@
+package integrations.kafka
+
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.hamcrest.Matchers
+import org.junit.Test
+import org.neo4j.function.ThrowingSupplier
+import org.neo4j.graphdb.Node
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.assertion.Assert
+import streams.serialization.JSONUtils
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+class KafkaEventSinkSimple: KafkaEventSinkBase() {
```

Comment:
Should this one have `IT` as the suffix of the class?

Reply:
I removed the `IT` from all the tests invoked by `KafkaEventSinkSuiteIT` because otherwise they trigger this configuration:

https://github.com/neo4j-contrib/neo4j-streams/blob/b597f09918a4de345286de558d38aaee6b5f693a/pom.xml#L250-L258

And they would be executed twice.
Should we define a naming convention for all the tests involved in a suite? Something like `<NAME>Suite`, or whatever else.